etc/systemd/system/proxysql.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=High Performance Advanced Proxy for MySQL
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/proxysql -c /etc/proxysql.cnf
+PIDFile=/var/run/proxysql/proxysql.pid
```

Comment:
Why putting a `PIDFile`? as [documentation](https://www.freedesktop.org/software/systemd/man/systemd.service.html#PIDFile=) say, it as to be used with `Type=forking`